src/SearchService.Models.Dto/Responses/SearchResultResponse.cs

```diff
@@ -0,0 +1,35 @@
+﻿using DigitalOffice.Models.Broker.Models.Department;
+using DigitalOffice.Models.Broker.Models.News;
+using DigitalOffice.Models.Broker.Models.Office;
+using DigitalOffice.Models.Broker.Models.Project;
+using DigitalOffice.Models.Broker.Models.User;
+using DigitalOffice.Models.Broker.Responses.Search;
+using LT.DigitalOffice.Models.Broker.Responses.Search;
+
+namespace LT.DigitalOffice.SearchService.Models.Dto.Response;
+
+public class SearchResultResponse
+{
+  public ISearchResponse<DepartmentSearchData> Department { get; set; }
```

Comment:
Maybe rename to DepartmentSearchResponse? In my opinion, the Department is not quite suitable for the answer, where the result is a list